packages/core/src/components/DataTable/DataTable.tsx

```diff
@@ -518,11 +310,10 @@ export class DataTable extends React.Component<DataTableProps & WithStylesProps,
             overscanRowCount={
               dynamicRowHeight && showAllRows ? expandedData.length : overscanRowCount!
             }
-            onRowClick={this.handleRowClick}
+            onRowClick={() => rowClickCallback}
```

Comment:
Should this be  `onRowClick={() => rowClickCallback()}`? And was it intentional to hide the onRowClick API from `react-virtualized`? It's [function signature](https://github.com/bvaughn/react-virtualized/blob/master/docs/Table.md) is `({ event: Event, index: number, rowData: any }): void` and I imagine at least some of that would be useful to the end users.

Reply:
^^^

I'd also keep `handleRowClick` to keep referential equality.